libsolidity/interface/ABI.cpp

```diff
@@ -44,14 +44,13 @@ Json::Value ABI::generate(ContractDefinition const& _contractDef)
 
 	for (auto it: _contractDef.interfaceFunctions())
 	{
-		if (
-			_contractDef.isLibrary() &&
-			(it.second->stateMutability() > StateMutability::View ||
-			anyDataStoredInStorage(it.second->parameterTypes() + it.second->returnParameterTypes()))
-		)
+		if (_contractDef.isLibrary() && (
```

Comment:
Why was the style wrong?

Reply:
The expression starting in line 49 was broken into two lines, but there was no newline after the opening parenthesis.
The following would also be ok:
```
		if (
			_contractDef.isLibrary() && (
				it.second->stateMutability() > StateMutability::View ||
				anyDataStoredInStorage(it.second->parameterTypes() + it.second->returnParameterTypes())
			)
		)
```

Reply:
Ok